platforms/common/src/main/resources/assets/dynamic_fps/lang/de_at.json

```diff
@@ -15,11 +21,41 @@
     "config.dynamic_fps.idle_time": "Untätigkeits-Timeout",
     "config.dynamic_fps.idle_time_tooltip": "Zeit in Minuten ohne Aktivität, bis die Untätig Konfiguration aktiviert wird, während das Spiel fokussiert ist.",
 
+    "config.dynamic_fps.idle_condition": "Untätigkeits-Bedingung",
+    "config.dynamic_fps.idle_condition_none": "Keine",
+    "config.dynamic_fps.idle_condition_on_battery": "Im Akkubetrieb",
+
+    "config.dynamic_fps.battery_tracker": "Akku-Tracker",
+    "config.dynamic_fps.battery_tracker_tooltip": "Schalte alle Akku-Features ein oder aus.",
+
+    "config.dynamic_fps.battery_tracker_switch_states": "Nutze den Auf Akku Status",
+    "config.dynamic_fps.battery_tracker_switch_states_tooltip": "Wechsel automatisch zum Auf Akku Status wenn dein Gerät auf Akku läuft.",
+
+    "config.dynamic_fps.battery_tracker_notifications": "Akku Benachrichtigungen",
+    "config.dynamic_fps.battery_tracker_notifications_tooltip": "Erhalte im Spiel Benachrichtigungen zum Akkustatus.",
+
+    "config.dynamic_fps.battery_indicator_condition": "Akkuanzeige Bedingung",
+    "config.dynamic_fps.battery_indicator_condition_disabled": "Deaktiviert",
+    "config.dynamic_fps.battery_indicator_condition_draining": "Während des Entladens",
+    "config.dynamic_fps.battery_indicator_condition_critical": "Kritisch / <= 10%",
```

Comment:
I'm not sure `<=` is clear to non-programmers, and `<` would be close enough anyway since it's a continuous value (i.e. 9.9% would be < 10%).

Reply:
I'm a bit late here, but Minecraft supports many advanced symbols from the Unicode table, including `≤`. A symbol that people are definitely familiar with from school.

Reply:
I changed it to just a < in both English and German, think that's fine :)

Reply:
I believe the unicode <= would be displayed in a different font as well, producing a jarring effect.